portal-ui/src/screens/Console/Tenants/TenantDetails/TenantDetails.tsx

```diff
@@ -297,6 +302,26 @@ const TenantDetails = ({
       });
   };
 
+  const loadPods = () => {
+    api
+      .invoke(
+        "GET",
+        `/api/v1/namespaces/${tenantNamespace}/tenants/${tenantName}/pods`
+      )
+      .then((result: IPodListElement[]) => {
+        for (let i = 0; i < result.length; i++) {
+          let currentTime = new Date().getSeconds();
+          result[i].time = niceDays(
+            (currentTime - parseInt(result[i].timeCreated)).toString()
+          );
+        }
+        setPods(result);
+      })
+      .catch((err) => {
+        console.log("Error in loading pods");
```

Comment:
Should we show the error alert here?

```suggestion
        setErrorSnackMessage("Error loading pods")
```